### PR TITLE
feature: added categories

### PR DESCRIPTION
**Pull request by omgaikwad**
feature: added categories
Preview: https://deploy-preview-5--gallant-pasteur-062544.netlify.app/pages/explore/explore.html

### CODE REVIEW

**Review by Shivanshsharma01 — APPROVED** [review]
clean code, well structure, good use of variables

File: index.html [hunk 1]
```diff
@@ -146,7 +194,11 @@ <h4>BREAKING BAD</h4>
           </div>
         </a>
         <!-- Trivia Card -->
-        <a class="card-vertical" id="trivia-card" href="#">
+        <a
+          class="card-vertical"
+          id="trivia-card"
```
Comment (prachi-sahani) [inline index.html]: id attribute for each element should have unique value throughout the page

Reference - https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id

**Review by prachi-sahani — COMMENTED** [review]
UI is very nice
Suggestion - include files related to PR only

File: index.html (hunk 1 above)
Comment (omgaikwad) [reply index.html]: Sure, I will change it to class

**Pull request merged**